README.md

```diff
@@ -1 +1,42 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+# hello-microverse![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> HELLO MICROVERSE PROJECT.
+
+
+## Getting Started
+
+**This is the first project created on microverse education by me.**
+
+
+## What was done on the project
+
+**created a  repository**
+**adding new branch**
+**adding .gitignore file**
+**set up linters for html and css**
+**create html, css and test(which should be ignored)**
+
+
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Dmambo)
+- Twitter: [@twitterhandle](https://twitter.com/kingibro345)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+
```

Comment:
**[Optional]**

- [ ] Kindly add license file in your project to make your project licensed. You can use these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to create an license file in your readme file :+1: Each project should be licensed if it has been created by you. :thinking:
- [ ] After adding your readme file it would be nice if you add your license file link in your readme file. You can see [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) to get an idea what i am talking about :blush: